src/components/migrationToV3.stories.mdx

```diff
@@ -0,0 +1,119 @@
+npm install --save amazon-chime-sdk-component-library-react amazon-chime-sdk-js
+```
+
+## What’s New
+
+- Updated props for `MeetingProvider`.
+- Updated `join` method props in `MeetingManager` component.
+
+### Meeting Provider prop change
+Meeting provider no longer takes `VideoDownlinkBandwidthPolicy`, `VideoUplinkBandwidthPolicy`, `simulcastEnabled`, `reconnectTimeoutMs`, and `keepLastFrameWhenPaused` as parameters.
+
+These properties can rather be updated by passing the `MeetingSessionConfiguration` via `meetingManager.join()` method. The following example uses `VideoDownlinkBandwidthPolicy` property as an example to showcase the change.
+
+If your code looks like this:
+```
+const videoDownlinkBandwidthPolicy = new MyVideoDownlinkBandwidthPolicy();
+const meetingConfig = {
+  videoDownlinkBandwidthPolicy,
+};
+
+<MeetingProvider {...meetingConfig}>
+  <MyApp />
+</MeetingProvider>
+
+const meetingManager = useMeetingManager();
+meetingManger.join({
+  data.JoinInfo.Meeting,
+  data.JoinInfo.Attendee
+});
+```
+
+change it to like this:
+```
+const videoDownlinkBandwidthPolicy = new MyVideoDownlinkBandwidthPolicy();
+<MeetingProvider>
+  <MyApp />
+</MeetingProvider>
+
+let meetingSessionConfiguration = new MeetingSessionConfiguration(data.JoinInfo.Meeting, data.JoinInfo.Attendee);
+meetingSessionConfiguration.videoDownlinkBandwidthPolicy = videoDownlinkBandwidthPolicy;
+
+const meetingManager = useMeetingManager();
+meetingManager.join({
+  meetingSessionConfiguration
+});
+```
+
+#### simulcastEnabled
+One thing to note is that the `MeetingProvider` props might not have the same mapping with `MeetingSessionConfiguration` properties.
+For exmaple, `simulcastEnabled` in `MeetingProvider` and the property in `MeetingSessionConfiguration` is different. The following code example should help remove any confusion.
+
+If your code looks like this:
+```
+const meetingConfig = {
+  simulcastEnabled: false,
+};
+
+<MeetingProvider {...meetingConfig}>
+  <MyApp />
+</MeetingProvider>
+
+const meetingManager = useMeetingManager();
+meetingManger.join({
+  data.JoinInfo.Meeting,
+  data.JoinInfo.Attendee
+});
+```
+
+change it to like this:
+```
+<MeetingProvider>
+  <MyApp />
+</MeetingProvider>
+
+let meetingSessionConfiguration = new MeetingSessionConfiguration(data.JoinInfo.Meeting, data.JoinInfo.Attendee);
+meetingSessionConfiguration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers = false;
+
+const meetingManager = useMeetingManager();
+meetingManager.join({
+  meetingSessionConfiguration
+});
+```
+
+### Meeting Manager `join` method prop change
+Meeting Manager `join` method paramters use the `MeetingJoinData` interface. In V3, this interface has been updated to include `MeetingSessionConfiguration`. 
+`meetingInfo` and `attendeeInfo` parameters are removed from the interface and are rather passed to the constructor of `MeetingSessionConfiguration`.
+
+If you code looks like this:
+```
+meetingManager.join({
+  data.MeetingInfo,
```

Comment:
My goal is to provide a more syntactically correct code example. The call to the meetings backend is returning data in a json object with `JoinInfo` hence I added it.

Reply:
Since, we do not implement the backend here and it is up to builders on how to implement it, we wont be knowing whether the response is under a certain key hence suggested so. Also, I just want the usage to be consistent with our examples elsewhere, no big concerns.